yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -631,65 +633,74 @@ def pixelize_aitoff(np.float64_t[:] theta,
         img = input_img
     # Okay, here's our strategy.  We compute the bounds in x and y, which will
     # be a rectangle, and then for each x, y position we check to see if it's
-    # within our theta.  This will cost *more* computations of the
-    # (x,y)->(theta,phi) calculation, but because we no longer have to search
-    # through the theta, phi arrays, it should be faster.
-    dx = 2.0 / (img.shape[0] - 1)
-    dy = 2.0 / (img.shape[1] - 1)
+    # within our Lambda.  This will cost *more* computations of the
+    # (x,y)->(Lambda,btheta) calculation, but because we no longer have to search
+    # through the Lambda, btheta arrays, it should be faster.
+    xw = bounds[1] - bounds[0]
+    yw = bounds[3] - bounds[2]
+    dx = xw / (img.shape[0] - 1)
+    dy = yw / (img.shape[1] - 1)
     x = y = 0
     for fi in range(nf):
-        theta_p = (theta[fi] + theta_offset) - PI
-        dtheta_p = dtheta[fi]
-        phi_p = (phi[fi] + phi_offset) - PI/2.0
-        dphi_p = dphi[fi]
+        Lambda_p = (azimuth[fi] + azimuth_offset) - PI
+        dLambda_p = dazimuth[fi]
+        btheta_p = PI/2.0 - (colatitude[fi] + colatitude_offset)
```

Comment:
Can you spare a moment to explain why this is different now?  It's the reverse of before, yes?

Reply:
yes. AFAICT, everywhere in the code base, we denote the _colatitude_ "theta". As the name suggests, _colatitude_ is the complementary angle to _latitude_ (so lat + coat = PI/2). On the main branch, the angle that's noted `phi` here effectively is the opposite of latitude. So I'm changing the name to `btheta` to reflect that it's neither the azimuth (phi) nor the colatitude (theta), and I'm changing its sign because... that's just a bug (I discovered it in testing when I saw a sphere being displayed upside down, with the north emisphere on the bottom half).